### PR TITLE
Avoid conflicts with native tab handlers

### PR DESCRIPTION
**Pull request by chcamiloam**
Fix for rgh issue 
https://github.com/refined-github/refined-github/issues/5903

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ export function indent(element: HTMLTextAreaElement): void {
 	} else {
 		insert(element, '\t');
 	}
+	event.stopImmediatePropagation()
```
Comment (fregante) [inline index.ts]: This is the wrong place, all tests are failing. Did you test it on Refined GitHub?

**Review by fregante — CHANGES_REQUESTED** [review]

**Comment by fregante** [comment]
PR titles should not describe the code changes but what they're doing for the user

File: index.ts (hunk 1 above)
Comment (chcamiloam) [reply index.ts]: I did, and everything is working correctly, let try something else :)

**Comment by fregante** [comment]
I will have to fix the tests before merging this. It looks like browsers changed the behavior

**Comment by fregante** [comment]
Please run tests locally before pushing because every commit here is broken one way or another

**Pull request merged**